app/assets/stylesheets/modules/_forms.css.scss

```diff
@@ -25,3 +25,17 @@
   margin: 1em 0 5em;
   padding-top: 1em;
 }
+
+input.required {
+  border: 1px solid darken(#edd1d1, 50%);
```

Comment:
@mghaught I know you mentioned highlighting required fields. I didn't see specifics around this in the style guide, so went with an existing color that I thought made sense. Let me know if you'd like me to change this.